BLL/Services/CandidateService.cs

```diff
@@ -39,9 +39,9 @@ public void DeleteCandidate(int id)
             _db.Save();
         }
 
-        public List<CandidateDTO> GetAllCandidatesWithStatuses()
+        public List<CandidateDTOForGetAll> GetAllCandidatesWithStatuses()
         {
-            var result = GetCandidatesWithStatusesInformation().ToList();
+            var result = GetAllCandidatesWithStatusesInformation().ToList();
             result.ForEach(x => x.SandboxCount = GetCountOfSandboxes(x.ID));
```

Comment:
result.ForEach(x => x.SandboxCount = GetCountOfSandboxes(x.ID));
 it's a bad practice (a lot of query to the database)